restclient/src/test/java/com/box/l10n/mojito/rest/client/VirtualAssetPerformanceTest.java

```diff
@@ -68,14 +68,15 @@ public void performance() throws RepositoryNotFoundException {
 
     logger.debug("virtual asset id: {}", virtualAsset.getId());
 
-    DateTime start = DateTime.now();
+    ZonedDateTime start = ZonedDateTime.now();
     //        createSourceStrings(virtualAsset);
     //        importTranslations(repository, virtualAsset);
 
     pullSourceString(virtualAsset, repository);
 
     pullTranslations(virtualAsset, repository);
-    logger.debug("total: {}", PeriodFormat.getDefault().print(new Period(start, DateTime.now())));
+    Duration duration = Duration.between(start, ZonedDateTime.now());
```

Comment:
I guess there is no equivalent of `PeriodFormat`? What was the old output looking like? Would using `duration.toString()` closer to the old output?

Reply:
java has a class called Period, but Period represents a date-based quantity, such as days, months, and years. Duration is similar, but represents a time-based quantity, such as hours, minutes, seconds, and nanoseconds.
The output is different, something like:
 joda.Period: 2 seconds and 8 milliseconds
 java.Duration: PT2.011S